presto-hive/src/main/java/io/prestosql/plugin/hive/HivePartitionManager.java

```diff
@@ -143,7 +142,7 @@ public HivePartitionResult getPartitions(SemiTransactionalHiveMetastore metastor
                     ImmutableList.of(new HivePartition(tableName)),
                     compactEffectivePredicate,
                     effectivePredicate,
-                    all(),
+                    TupleDomain.all(),
```

Comment:
We could have static analysis for this

Reply:
We ban certain static imports in Checkstyle in Airbase, but we'd need a way to provide an additional Checkstyle ruleset for this project. Unfortunately, it looks like `maven-checkstyle-plugin` only allows a single `configLocation`.